server/liveblog/themes/themes_assets/angular/theme.json

```diff
@@ -89,8 +89,8 @@
     ],
     "scripts": [
         "https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js",
-        "https://ajax.googleapis.com/ajax/libs/angularjs/1.5.11/angular.min.js",
-        "https://cdnjs.cloudflare.com/ajax/libs/angular-resource/1.5.11/angular-resource.min.js",
+        "https://ajax.googleapis.com/ajax/libs/angularjs/1.5.11/angular.js",
+        "https://cdnjs.cloudflare.com/ajax/libs/angular-resource/1.5.11/angular-resource.js",
```

Comment:
Please move this back to `.min.js` versions as `scripts` is for production.

Reply:
done